cloudinit/sources/DataSourceOracle.py

```diff
@@ -1,22 +1,19 @@
 # This file is part of cloud-init. See LICENSE file for license information.
 """Datasource for Oracle (OCI/Oracle Cloud Infrastructure)
 
-OCI provides a OpenStack like metadata service which provides only
-'2013-10-17' and 'latest' versions..
-
 Notes:
- * This datasource does not support the OCI-Classic. OCI-Classic
-   provides an EC2 lookalike metadata service.
- * The uuid provided in DMI data is not the same as the meta-data provided
+ * This datasource does not support OCI Classic. OCI Classic provides an EC2
+   lookalike metadata service.
+ * The UUID provided in DMI data is not the same as the meta-data provided
    instance-id, but has an equivalent lifespan.
  * We do need to support upgrade from an instance that cloud-init
    identified as OpenStack.
- * Both bare-metal and vms use iscsi root
- * Both bare-metal and vms provide chassis-asset-tag of OracleCloud.com
+ * Bare metal instances use iSCSI root, virtual machines do not
+ * Both bare-metal and VMs provide chassis-asset-tag of OracleCloud.com
```

Comment:
The edits here are just kind of odd.  You seem to prefer 'Bare metal' to 'bare-metal' on line 11, but then back to 'bare-metal' on line 12.  and then you replaced 'vms' with 'virtual machines' on line 14 -> 11, but then capitalized VMs on line 12.

only really mentioning because i had other comments.

Reply:
Yep, no idea why I did that, will fix.